website/static/js/projectorganizer.js

```diff
@@ -644,13 +643,17 @@ function _poToggleCheck(item) {
 function _poResolveIcon(item) {
     var icons = iconmap.projectIcons;
     var componentIcons = iconmap.componentIcons;
+    var projectIcons = iconmap.projectIcons;
     var viewLink = item.data.urls.fetch;
     function returnView(type, category) {
         var iconType = icons[type];
         if (type === 'component' || type === 'registeredComponent') {
             iconType = componentIcons[category];
         }
-        if (type === 'registeredComponent') {
+        else if (type === 'project' || type === 'registeredProject') {
```

Comment:
That said, @caileyfitz noted that the icon for a registered project is different from the regular project icon, which is different from all the other categories (which use a lighter icon for registrations). Do we need a different icon for registered projects?